Gemfile

```diff
@@ -48,6 +48,7 @@ gem 'bootsnap', require: false
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
   gem 'debug', platforms: %i[mri windows]
+  gem 'rails-controller-testing'
   gem 'rspec-rails'
 end
 
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you elevate your project's appeal with Future Features 🌟. In the competitive world of software development, staying ahead of the curve is essential. To truly shine and seize massive opportunities in the job market, consider enhancing your project's readme file with exciting, game-changing features. 🚀💡🔗🏆